src/assets/content/WhatsNew.js

```diff
@@ -0,0 +1,36 @@
+export default {
+    blogListItems: [
+        {
+            id: '5',
+            date: '07/21/2023:',
+            title: 'Jazz up your ggplots!',
+            url: 'https://waterdata.usgs.gov/blog/ggplot-jazz/',
+            img_src: 'https://waterdata.usgs.gov/blog/static/ggplot-extensions/ggplot-extensions-banner.png',
+            img_alt: 'Musical notes drawn with the center of the notes containing data viz example'
+        },        
```

Comment:
Moving forward we can include all of this information for each entry. Within the component we can just show the first one, or maybe down the line we'll come up with a way to feature more than one.